pkg/minishift/hostfolder/sshfs_hostfolder.go

```diff
@@ -33,7 +35,7 @@ import (
 )
 
 const (
-	keyFile = "/home/docker/.ssh/id_rsa"
+	KeyFile = "/home/docker/.ssh/id_rsa"
```

Comment:
Why export this?

Reply:
Ah, was exporting to be used in another file. Not needed now, reversing now.

Reply:
Done